pkg/cli/cli_test.go

```diff
@@ -170,3 +171,14 @@ func newTestIOStreams() (IOStreams, *bytes.Buffer, *bytes.Buffer) {
 		ErrOut: errOut,
 	}, in, out
 }
+
+func cvdOutput(serviceURL, host, cvd string) string {
+	output := CVDOutput{
+		BaseURL: serviceURL + "/v1",
+		Host:    "foo",
+		CVD: &hoapi.CVD{
+			Name: "cvd-1",
+		},
+	}
+	return output.String() + "\n"
```

Comment:
nit: if you had function that lists cvds and returns `[]CVDOutput` you wouldn't need to play with these string manipulations.

Reply:
agree, there's room for refactoring here which I'd like to do in a different PR as it would be reused in other parts of the code.

Reply:
went ahead and removed the string manipulations in the tests.

Reply:
done